changelog/next/features/3669--put-as-asource.md

```diff
@@ -0,0 +1,2 @@
+The `put` operator is now a source operator as well, which allows for
+introducing one event into a pipeline with known values.
```

Comment:
@jachris and I discussed on Thursday that this is quite the breaking change to make because of implicit sources, and that we'd rather introduce a new operator for this. I agree with this fundamentally. @mavam, do you have any naming ideas?